bot/cogs/information.py

```diff
@@ -104,36 +100,18 @@ def __init__(self, bot: Bot):
         features = ", ".join(ctx.guild.features)
         region = ctx.guild.region
 
-        # How many of each type of channel?
         roles = len(ctx.guild.roles)
-        channels = ctx.guild.channels
-        text_channels = 0
-        category_channels = 0
-        voice_channels = 0
-        for channel in channels:
-            if type(channel) == TextChannel:
-                text_channels += 1
-            elif type(channel) == CategoryChannel:
-                category_channels += 1
-            elif type(channel) == VoiceChannel:
-                voice_channels += 1
+        member_count = ctx.guild.member_count
+
+        # How many of each type of channel?
+        channels = Counter({TextChannel: 0, VoiceChannel: 0, CategoryChannel: 0})
+        for channel in ctx.guild.channels:
+            channels[channel.__class__] += 1
```

Comment:
I just realised the `Counter` is not being taken advantage of. The for loop can be done away with:

```py
channels = Counter((c.type for c in guild.channels))
```

The same can be done for the the statuses.